bids/modeling/statsmodels.py

```diff
@@ -343,6 +342,14 @@ def _build_groups(objects, group_by):
         entities = [obj.entities for obj in objects]
         df = pd.DataFrame.from_records(entities)
 
+        # Load participants.tsv as entities
+        if self.level == 'dataset':
```

Comment:
This seems wrong to make dependent on level. Why would we not care about these variables in all cases?

Reply:
Agree, although I think we need to load the relevant variables only on some levels? Or should we load all matching ones at any level?

Does it make sense to load participant level variables at the `subject `level model?